src/framebuffer.ml

```diff
@@ -419,6 +425,7 @@ let filled_polygon (points : (int * int) list) (col : int) (buffer : t) =
 (* ----- *)
 
 let draw_char (x : int) (y : int) (f : Font.t) (c : char) (col : int) (buffer : t) : int =
+  buffer.dirty <- true;
```

Comment:
Not necessary here I think, as this calls `pixel_write` and that should set the buffer.